glyphs2fontir/src/source.rs

```diff
@@ -168,4 +191,40 @@ mod tests {
             .collect::<HashSet<String>>();
         assert_eq!(HashSet::from(["hyphen".to_string()]), changed);
     }
+
+    #[test]
+    fn access_glyph_name_string() {
+        let mut glyph = HashMap::new();
+        glyph.insert(
+            "glyphname".to_string(),
+            Plist::String("space".to_string(), 0..1),
+        );
+        assert_eq!(
+            "space",
+            glyph_name(
+                Path::new("f.glyphs"),
+                &glyph,
+                &(1usize..12usize),
+                "I'm a plist"
```

Comment:
wait, why is this "I'm a plist"? Shouldn't this be something like `"glyphname = space"`?

Reply:
ah, it's unused if glyph["glyphname"] is a Plist::String.. still it's confusing

Reply:
I will update to something clearer, e.g. "Not used by test"